Add new target FICF4

ficueu opened the pull request:
Hello,
i made my own FC and i would like to add my target to releases.

[FicF4.zip](https://github.com/betaflight/betaflight/files/2238811/FicF4.zip)

wind0r commented:
This PR doesnt contain any new Target and also is against the wrong branch

mikeller commented:
@ficueu:
- there is no new target in your pull request;
- the target definition in the zip file you attached is just a copy of the FURYF4 target with a changed name - this will not be made into a new target, please use FURYF4, closing this.

mikeller commented:
@jflyper: A cursory glance did not show any functional differences, except for additions / drops. What did you spot?

ficueu commented:
@jflyper 
True, camera control works only on software PWM, but it works perfectly. LEDs working great. Hardware is most based on KIWI. I will order new PCBs and I try to change pin for camera control with other timer.

I'm new on the github and this is my first pull request, sorry for troubles.

mikeller commented:
@ficueu: What are the differences to the KIWIF4, and what is the rationale for having them? Every new target creates considerable effort for maintenance, and only reduces the amount of user feedback per target, increasing the chance of users ending up with broken releases for their hardware. Considering this, it would be appreciated if you changed your target to match the KIWIF4 target definition.

ficueu commented:
@mikeller KIWIF4 is no longer available to buy. Main differences in hardware: changed LED pin, added I2C, added UART4, added camera control, removed SD. In future the STM will be changed to F722.

jflyper commented:
@ficueu Please note the software PWM for camera control is no longer available.

mikeller commented:
The fact that the KIWIF4 is no longer available does not mean that we will drop support for it, as this would leave current owners stuck with old firmware, so the target for this will stick around.

Can I suggest that you change the LED pin back, and we can add support for I2C and UART4 to the existing target, making it work for the KIWIF4 and your board?

Once you move to F722 you will need a new target anyway, since the two aren't compatible in terms of features.

ficueu commented:
@jflyper 
For hardware PWM the camera control should use one available free timer?

@mikeller 
KIWIF4 has some issues with LEDstrip pin because of available timers (I'm not sure, probably it uses motor timer).

mikeller commented:
@ficueu: Ah, it is the LED_STRIP pin that you are referring to. That is a known issue with KIWIF4, so we can move this on the existing target definition without making things worse for owners of this board.
